README.md

```diff
@@ -82,85 +82,118 @@ To deploy this solution, perform the follow steps:
+    This setup script does the following:
+
+    - Validate software dependencies
+    - Enable the required APIs defined in `project_apis.txt`
+    - Enable the required IAM roles on the service account you'll use to deploy Terraform resources, defined in `persona_roles_DEPLOYER.txt`
+    - Set up the OAuth consent screen (brand) required for IAP. We recommend you create this resource using a user identity instead of a service account. This approach helps avoid problems related to [support_email ownership](https://cloud.google.com/iap/docs/programmatic-oauth-clients#:~:text=the%20user%20issuing%20the%20request%20must%20be%20an%20owner%20of%20the%20specified%20support%20email%20address) and [destroying a terraform-managed Brand resource](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_brand).
+    - Enables the required IAM roles used for underlying Cloud Build processes
+    - Authenticate [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials) with your service account credentials to be used by Terraform.
+    - Triggers a pop-up dialog box: 'Sign in with Google' prompting you to authenticate the Google Auth Library. Follow the directions to authenticate with your user account, which will then configure Application Default Credentials (ADC) using the impersonated service account credentials to be used by Terraform.
+
+1.  Create a terraform.tfvars file with the following variables:
+      | Terraform variables         | Description                                                                                          |
+      | --------------------------- | -----------------------------------------------------------------------------------------------------|
+      | project_id                  | Your Google Cloud project ID.                                                                        |
+      | region                      | The desired region for deploying single-region resources (e.g., "us-central1", "europe-west1").      |
+      | vertex_ai_data_store_region | The multiregion for your Agent Builder Data Store, the possible values are ("global", "us", or "eu").|
+      | docai_location              | Sets the location for Document AI                                                                    |
+      | webui_domains               | Your domain name for Web UI access (e.g., ["webui.example.com"])                                     |
+      | iap_access_domains          | List of domains granted for IAP access to the Web UI (e.g., ["domain:example.com"])                  |
+
+1.  (Optional) By default, the Terraform script creates a new VPC network in the same project as other resources. You can use an existing VPC network instead by configuring the following optional terraform variables.
+      | Terraform variables                 | Description                                    | 
```

Comment:
Line 118 introduced trailing whitespace that causes issues with Lint.

You can fix this and similar issues by running:
```
export LINTER_CONTAINER_FIX_MODE=true
scripts/lint.sh
```

Or, I configure my IDE with a devcontainer so that it will automatically configure the built-in linters to catch and fix this on every save.

Reply:
Ok, thanks!